usr/src/uts/armv8/rpi4/os/rpi4.c

```diff
@@ -317,3 +317,71 @@ plat_set_cpu_supp_freqs(cpu_t *cp)
 	cpupm_set_supp_freqs(cp, supp_freqs,
 	    sizeof (supp_freqs) / sizeof (supp_freqs[0]));
 }
+
+int
+plat_get_temperature(uint_t sens_id)
```

Comment:
Why did this end up in the platmod, and not the sensors driver?

Reply:
There is no good reason, at least for now (if ever...). Moved it to the sensors driver where it belongs to.